README.md

```diff
@@ -20,6 +20,7 @@ The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.
     * `LOGDNA_HOSTNAME`: Alternative Host Name *(Optional)*  
     * `LOGDNA_TAGS`: Comma-separated Tags *(Optional)*  
     * `LOGDNA_URL`: Custom Ingestion URL *(Optional)*
+    * `RAW_EVENT_MESSAGE`: Setting `line` to Raw `event.message` *(Optional)*
```

Comment:
Can this just be a `bool` instead of a string you need to parse later?

Reply:
@matt-march, environment variable can't be `bool`

Reply:
Hmm, alright. Just thinking about trying to make the parsing below easier. Would be good to at least give an indication of what are the expected/acceptable values.

Reply:
@matt-march, updated in the `README` by adding a small explanation